stacks/argocd/deploy-local.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -e
+
+ROOT_DIR=$(git rev-parse --show-toplevel)
+
+# create namespace
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: argocd
+EOF
+
+# set kubectl namespace
+kubectl config set-context --current --namespace=argocd
+
+# deploy argocd
+kubectl apply -f "$ROOT_DIR"/stacks/argocd/yaml/argocd.yaml
```

Comment:
This should point to your src YAML: `"$ROOT_DIR"/src/argocd/1.2.5/argocd.yaml`

Reply:
Corrected, I hadn't saved a few things in VS Code *facepalm* 🤦‍♀️